main.cpp

```diff
@@ -0,0 +1,297 @@
+#include <iostream>
+#include<conio.h>

```

Comment:
Avoid using conio.h as it is not a header file used in GNU standards. Basically, it's a library which helps your code while executing in TurboC.

Reply:
okay